tests/basic/storage_pool_test.go

```diff
@@ -31,7 +31,7 @@ import (
 
 const (
 	replicationUpdateTimeout = 4 * time.Hour
-	poolResizeTimeout        = time.Minute * 360
+	poolResizeTimeout        = time.Minute * 60
```

Comment:
Actually In longevity setup where pool size become large there it could take more time !!

Reply:
Even for resize disk, not add disk? Add disk has a separate timeout.

Is there a way we could change the timeout based on if it's longevity or not? The goal is to keep tests from stalling for a really long time before failing

Reply:
@Pure-AdamuKaapan can you raise a PTX regarding this for me.. Ill make changes to different timeout for drive add, add-disk and resize-disk. need to go through the tests and see they have appropriate time out

Reply:
Sounds good, thanks so much! I'll close this PR for now then.

Reply:
https://portworx.atlassian.net/browse/PTX-17782